homework/App.js

```diff
@@ -16,14 +16,26 @@ class App {
     // 1. Create the fixed HTML elements of your page
     // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
 
-    const root = document.getElementById('root');
-
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
-
+    this.root = document.getElementById('root');
+    this.header = Util.createAndAppend('header', this.root);
+    Util.createAndAppend('label', this.header, { text: 'HYF Repositories' });
+    this.select = Util.createAndAppend('select', this.header);
+    Util.createAndAppend('div', this.root, {
+      id: 'container',
+    });
     try {
       const repos = await Util.fetchJSON(url);
       this.repos = repos.map(repo => new Repository(repo));
-      // TODO: add your own code here
+      repos.forEach((repository, index) => {
```

Comment:
you have to choose either using `this.repos` or `repos` variable, I think it would be more usefull using the `this.repos` in case of modifing it somehow to keep track on it in the whole program, like sorting or deleting a field or...etc